src/main/java/org/cascadebot/cascadebot/data/managers/PlaylistManager.java

```diff
@@ -41,17 +50,27 @@ public static void savePlaylist(Playlist playlist) {
         CascadeBot.INS.getDatabaseManager().runAsyncTask(database -> {
             database.getCollection(COLLECTION, Playlist.class).insertOne(
                     playlist,
-                    new DebugLogCallback<>("Inserted new playlist with ID: " + playlist.getPlaylistID())
+                    new DebugLogCallback<>("Inserted new playlist with ID: " + playlist.getPlaylistId())
             );
         });
     }
 
     public static void replacePlaylist(Playlist playlist) {
         CascadeBot.INS.getDatabaseManager().runAsyncTask(database -> {
             database.getCollection(COLLECTION, Playlist.class).replaceOne(
-                    eq("_id", playlist.getPlaylistID()),
+                    eq("_id", playlist.getPlaylistId()),
                     playlist,
-                    new DebugLogCallback<>("Replaced Playlist with ID: " + playlist.getPlaylistID())
+                    new DebugLogCallback<>("Replaced Playlist with ID: " + playlist.getPlaylistId())
+            );
+        });
+    }
+
+    public static void deletePlaylistById(String id) {
+        CascadeBot.INS.getDatabaseManager().runAsyncTask(database -> {
+            ObjectId playlistId = new ObjectId(id);
+            database.getCollection(COLLECTION, Playlist.class).deleteOne(
+                    eq("_id", playlistId),
+                    new DebugLogCallback<>("Deleted playlist with ID: " + playlistId)
```

Comment:
1. change everything to lowercase Id
2. add a log with uppercase ID
3. profit...

Reply:
No

Reply:
/\